flake.nix

```diff
@@ -93,6 +94,7 @@
               libgbm
               pango
               wayland
+              gtk4
```

Comment:
why do we need to pull gtk4 in?

Reply:
`rust-analyzer` with vscode complained with that but i believe its fhs packaging issue with vscode tho, i could remove that but bumping the flake is essential since rust crate dependencies uses newer libraries

Reply:
Removed changes from `flake.nix`